### ISSUE TITLE
Support `\rm`

### ISSUE THREAD

**Issue by GraemeWatt** [HDembinski/unicodeitplus#5]
Although `\rm` (roman) is deprecated, it is still often present in older LaTeX encodings, particularly for subscripts and superscripts like `$x_{\rm Bj}$`.  Similarly for `\it` (italics) and `\bf` (bold).  It would be good to support these commands in `unicodeitplus`.

**Comment by HDembinski** [comment]
I agree, I have to extend the parser to support constructs like this`{some text \it more text}`.
